### ISSUE TITLE
Replace in buffer shows changes in project find results tab

### ISSUE THREAD

**Issue by Ben3eeE** [atom/find-and-replace#699]
### Prerequisites

* [X] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#check-if-the-problem-shows-up-in-safe-mode)?
* [X] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [X] Did you check the [debugging guide](flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [X] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [X] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [X] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-bug-report) or [enhancement request](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion), see the `CONTRIBUTING` guide.

### Description

When doing a project find and replace the project find results tab shows the changes before they occur, original text being red and new text being green.
The same visual thing happens for every file in the project find and replace tab when you do a replace in current buffer with the project find results tab open. Regardless if the current buffer search matches the project search.

### Steps to Reproduce

1. Find some text in current project, project find results tab opens
2. Add some text to replace in current project, original text gets marked as red and new text as green, this is ok.
3. With the project find results tab open, for example in the right pane do a buffer find and replace in the left pane.
4. The project find results tab shows the changes red/green text for every result in the project find results when adding text to replace in current buffer. Even if the current buffer search does not match the project find results search.

See image replace in project working as expected:
![replace in project](https://cloud.githubusercontent.com/assets/1058982/14341926/10cdcba4-fc96-11e5-817e-4821daffbfd7.PNG)
See image replace in buffer with replace in project tab open:
![replace in buffer](https://cloud.githubusercontent.com/assets/1058982/14341935/24df0536-fc96-11e5-8e4c-e460632047e5.PNG)

**Expected behavior:** Project find results to not be affected by the value of the replace in current buffer field, or maybe to only show actual changes that will be made and not mark everything with green/red text.

**Actual behavior:** Project find results tab gets affected by the value of replace in current buffer.

### Versions

1.6.X
Windows 7

**Comment by simurai** [comment]
Yeah, feels like a bug. When switching to **buffer** find+replace, the **project find results** shouldn't be updated.

**Comment by Ben3eeE** [comment]
I agree, seems there are two different issues. One that is a bug and one that is an enhancement.

**Bug:** Even if you search for something else in the buffer the project find results updates.
![find and replace search](https://cloud.githubusercontent.com/assets/1058982/19673861/06d7f908-9a84-11e6-9a0a-08c6b5b05c81.gif)

**Enhancement:** Only update the buffer that you are replacing in when doing a buffer find and replace with the project find results open.

**Comment by simurai** [comment]
What would be the better solution?

1. Only update "Project Find Results" when switching back to the Project panel.
2. Only update "Project Find Results" when doing a new project search.

**Comment by Ben3eeE** [comment]
I think 1 would be the better solution.
